vault/templates/install.sh.tpl

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env bash
-set -e
-
-# Time before the kernel kills a connection in CLOSE_WAIT
-# reduced to 5 minutes as Vault keeps a lot of connections
-echo 'net.ipv4.tcp_keepalive_time = 300' >> /etc/sysctl.conf
```

Comment:
out of interest, for future proofing does need to also include an IPv6 network or are we not planning on using IPv6?